ovos_plugin_manager/utils/audio.py

```
@@ -0,0 +1,646 @@
+            startupinfo=startup_info,
+        )
+        flac_data, stderr = process.communicate(wav_data)
+        return flac_data
+
+    def get_np_int16(self, convert_rate=None, convert_width=None) -> Array:
+        """
+        Produce a NumPy int16 array containing the audio samples.
+        
+        Parameters:
+            convert_rate (int, optional): Target sample rate in Hz to convert to before extraction.
+            convert_width (int, optional): Target sample width in bytes (1–4) to convert to before extraction.
```

Ruff (0.14.10)

Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)